docs/references/protocol/ledger-data/ledger-entry-types/directorynode.md

```diff
@@ -56,19 +55,42 @@ There are two kinds of Directories:
+         ],
+         "LedgerEntryType": "DirectoryNode",
+         "NFTokenID": "000800009988C43C563A7BB35AF34D642990CDF089F11B445EB3DCCD00000132",
+         "RootIndex": "CC45A27DAF06BFA45E8AFC92801AD06A06B7004DAD0F7022E439B3A2F6FA5B5A",
+         "index": "CC45A27DAF06BFA45E8AFC92801AD06A06B7004DAD0F7022E439B3A2F6FA5B5A"
+      },
+      "status": "success",
+      "validated": false
+   }
+}```
+{% /tab %}
+
 {% /tabs %}
 
 ## {% $frontmatter.seo.title %} Fields
 
 | Name                | JSON Type | [Internal Type][] | Required? | Description |
 |:--------------------|:----------|:------------------|:----------|:------------|
-| `ExchangeRate`      | String    | UInt64            | No        | (Offer Directories only) **DEPRECATED**. Do not use. |
+| `ExchangeRate`      | String    | UInt64            | No        | (Offer directories only) **DEPRECATED**. Do not use. |
 | `Flags`             | Number    | UInt32            | Yes       | A bit-map of boolean flags enabled for this object. Currently, the protocol defines no flags for `DirectoryNode` objects. The value is always `0`. |
-| `Indexes`           | Array     | Vector256         | Yes       | The contents of this Directory: an array of IDs of other objects. |
-| `IndexNext`         | Number    | UInt64            | No        | If this Directory consists of multiple pages, this ID links to the next object in the chain, wrapping around at the end. |
-| `IndexPrevious`     | Number    | UInt64            | No        | If this Directory consists of multiple pages, this ID links to the previous object in the chain, wrapping around at the beginning. |
-| `LedgerEntryType`   | String    | UInt16            | Yes       | The value `0x0064`, mapped to the string `DirectoryNode`, indicates that this object is part of a Directory. |
-| `Owner`             | String    | AccountID         | No        | (Owner Directories only) The address of the account that owns the objects in this directory. |
+| `Indexes`           | Array     | Vector256         | Yes       | The contents of this directory: an array of IDs of other objects. |
+| `IndexNext`         | Number    | UInt64            | No        | If this directory consists of multiple pages, this ID links to the next object in the chain, wrapping around at the end. |
+| `IndexPrevious`     | Number    | UInt64            | No        | If this directory consists of multiple pages, this ID links to the previous object in the chain, wrapping around at the beginning. |
+| `LedgerEntryType`   | String    | UInt16            | Yes       | The value `0x0064`, mapped to the string `DirectoryNode`, indicates that this object is part of a directory. |
+| `NFTokenID`         | String    | Hash25            | Yes       | ID of the NFT in a buy or sell offer. |
+| `Owner`             | String    | AccountID         | No        | (Owner and NFT Offer Directories only) The address of the account that owns the objects in this directory. |
```

Comment:
I dont think Owner is assciated with NFT Offer directories, should be just (Owner Directories only)